unity/EditorPlugin/PluginEntryPoint.cs

```diff
@@ -310,22 +317,22 @@ private static void SetupAssemblyReloadEvents()
                 ourLogger.Info("UnlockReloadAssemblies");
                 EditorApplication.UnlockReloadAssemblies();
               }
+
               PlayModeSavedState = newPlayModeState;
             }
           });
         }
       };
 
-      AppDomain.CurrentDomain.DomainUnload += (sender, args) =>
+      // TODO: Why are we doing this? Why reset play mode when the AppDomain is being unloaded?
```

Comment:
https://github.com/JetBrains/resharper-unity/issues/520

https://github.com/JetBrains/resharper-unity/pull/609

Reply:
Thanks. I'll add a comment for context

Reply:
Done